src/zm_monitor.cpp

```diff
@@ -1120,7 +1121,9 @@ bool Monitor::connect() {
 #if HAVE_LIBCURL    //janus setup. Depends on libcurl.
     if (janus_enabled && (path.find("rtsp://") !=  std::string::npos)) {
       if (add_to_janus() != 0) {
-        Warning("Failed to add monitor stream to Janus!");
+        if (add_to_janus() != 0) {
```

Comment:
What's this all about? Just trying twice?  Maybe comment why so that someone doesn't remove it later.

Reply:
Hoping to eventually replace this with something better, but noted. Next PR will either include a comment, or a better solution.